src-docs/src/views/nav_drawer/nav_drawer_example.js

```diff
@@ -43,9 +43,11 @@ export const NavDrawerExample = {
               Providing a <EuiCode>flyoutMenu</EuiCode> prop on the{' '}
               <EuiCode>listItems</EuiCode> object of an{' '}
               <EuiCode>EuiNavDrawerGroup</EuiCode> will result in that link
-              opening a secondary menu. Note that this will also override the{' '}
-              <EuiCode>onClick</EuiCode> event. See sample data in the Demo JS
-              tab.
+              opening a secondary menu. For more details about other props
+              available for <EuiCode>listItems</EuiCode> object, please refer to{' '}
+              <EuiCode>EuiListGroupItem</EuiCode>. Note that this will also
+              override the <EuiCode>onClick</EuiCode> event. See sample data in
+              the Demo JS tab.
```

Comment:
I would actually move your added sentence to the last sentence since it breaks up the comment about `flyoutMenu` vs `onClick`.
```suggestion
              opening a secondary menu. Note that this will also
              override the <EuiCode>onClick</EuiCode> event. 
              For more details about other props
              available for the <EuiCode>listItems</EuiCode> object, please refer to{' '}
              <EuiCode>EuiListGroupItem</EuiCode>.
```

Reply:
Ok, got it